client/src/components/RequestModal.js

```diff
@@ -231,8 +230,10 @@ export default class RequestModal extends Component {
                         primary
                         onClick={this.submitRequest}
                         disabled={
-                                (this.state.topicValue === '') || 
-                                (this.state.availabilityValue === '')
+                                ((this.state.topicValue === '' || 
+                                this.state.availabilityValue === '') &&
+                                (this.state.topicValue === '' ||
+                                this.state.note === ''))
```

Comment:
This was discussed in meeting - a lack of time should not prevent a request from being made, but a student needs to leave a note if they don't provide a time